MastodonSDK/Sources/MastodonCore/Extension/CoreDataStack/Status+Property.swift

```diff
@@ -51,14 +51,28 @@ extension Mastodon.Entity.Status {
         guard let mediaAttachments = mediaAttachments else { return [] }
         
         let attachments = mediaAttachments.compactMap { media -> MastodonAttachment? in
-            guard let kind = media.attachmentKind,
-                  let meta = media.meta,
-                  let original = meta.original,
-                  let width = original.width,       // audio has width/height
-                  let height = original.height
+            guard let kind = media.attachmentKind
             else { return nil }
-            
-            let durationMS: Int? = original.duration.flatMap { Int($0 * 1000) }
+
+            let width: Int;
+            let height: Int;
+            let durationMS: Int?;
+
+            if let meta = media.meta,
+               let original = meta.original,
+               let originalWidth = original.width,
+               let originalHeight = original.height {
+                width = originalWidth               // audio has width/height
+                height = originalHeight
+                durationMS = original.duration.map { Int($0 * 1000) }
```

Comment:
What's the reason you changed `durationMS = ...`? What does it have to do with `width` and `height`?

Reply:
I suggested this change in https://github.com/mastodon/mastodon-ios/pull/805#discussion_r1054965829 partially because I didn't see that it was copy-pasted from existing code.

Reply:
Okay, I see. But I don't really get it. Why not something like

```swift
let width = media.meta?.original?.width ?? 32
let height = media.meta?.original?.height ?? 32
// duration
```

? `durationMS` would be set regardless of `width`/`height`? Or doesn't this make sense?

Reply:
Oh I see, that part makes more sense now — I agree that it shouldn't be controlled by the `width`/`height` properties (and in fact it will probably break audio attachments if the duration isn’t set properly)